src/main/java/RWSet.java

```diff
@@ -1,12 +1,13 @@
 import java.util.Set;
 import java.util.ArrayList;
 import java.util.HashSet;
+import java.util.Random;
 
 /**
  * Remove Wins Set
  */
-public class RWSet<T extends Object> extends CRDT {
-    private static final long serialVersionUID = 1L;
+public class RWSet<T> extends CRDT {
+    private static final long serialVersionUID = new Random().nextLong();
```

Comment:
Ah, sorry!  I should have been more specific.  This will definitely break.  You need to set it to a random constant *in the source*, not generate a new random number at runtime.  The important thing is that both sides have the same number, and it's a lot easier to debug when that number is unique to the particular class.